en/SettingsView/General.md

```diff
@@ -1,49 +1,110 @@
 # General Settings (Settings View)
 
-The general settings (**SettingsView > General Settings**) are the main place for application-level configuration. Settable values include: display units, autoconnection devices, video display and storage, RTK GPS, and other miscellaneous settings.
+The general settings (**SettingsView > General Settings**) are the main place for application-level configuration. Settable values include: display units, autoconnection devices, video display and storage, RTK GPS, brand image, and other miscellaneous settings.
 
 > **Note** Values are settable even if no vehicle is connected. Settings that require a vehicle restart are indicated in the UI.
 
-![](../../images/settings/settings_view_general.jpg)
+![SettingsView - Full General Tab](../../images/settings/settings_view_general.jpg)
 
 
 ## Units
 
-This section defines the display units used for: distance, area and speed.
+This section defines the display units used in the application.
 
 ![Units settings](../../images/settings/settings_view_general_units.jpg)
 
+The settings are:
+- **Distance**: Meters | Feet
+- **Area**: SquareMetres | SquareFeet | SquareKilometers | Hectares | Acres |SquareMiles
+- **Speed**: Metres/second | Feet/second | Miles/hour | Kilometres/hour | Knots
+- **Temperature**: Celsius | Fahrenheit
 
 ## Miscellaneous
 
-This section defines a number of miscellaneous settings, related to (non exhaustively): font sizes, colour schemes, map providers, map types, telemetry logging, audio output, low battery announcement levels, default mission altitude, [virtual joysticks](../SettingsView/VirtualJoystick.md), mission autoloading, default file save path etc.
+This section defines a number of miscellaneous settings, related to (non exhaustively): font sizes, colour schemes, map providers, map types, telemetry logging, audio output, low battery announcement levels, default mission altitude, [virtual joysticks](../SettingsView/VirtualJoystick.md), mission autoloading, default application file load/save path etc.
 
 ![Miscellaneous settings](../../images/settings/settings_view_general_miscellaneous.jpg)
 
+The settings are:
+- <span id="colour_scheme"></span>**Color Scheme**: Indoor (Dark) | Outdoor (Light)
+- **Map Provider**: Bing | Google | Statkart | Eniro
+- **Map Type**: Hybrid | Street | Satellite
+- **Stream GCS Position**: Never | Always | When in Follow Me flight mode.
+- **Font Size**: Font size across app (Requires restart)
+- **Mute all audio output**: Turns off all audio output. 
+- <span id="autosave_log"></span>**Save telemetry log after each flight**: Logs automatically saved to the *File Save Path* ([below](#file_save_path)) after flight. 
+- **Save telemetry log even if vehicle was not armed**: 
```

Comment:
SO when would this trigger? I THINK that normally logs are collected from arming point, and what this does is start collecting as early as possible. It would presumably still save on disarming?

Essentially this is more "Log early" than anything else?

Reply:
Starts when a vehicle connects to QGC. Stops when the last vehicle disonnects from QGC.